cli/js/compiler.ts

```diff
@@ -673,7 +632,7 @@ function buildSourceFileCache(
   }
 }
 
-interface EmmitedSource {
+interface EmittedSource {
```

Comment:
Yikes.

Reply:
Those are all private interfaces not visible to end users